.travis.yml

```diff
@@ -1,5 +1,9 @@
 sudo: false
 language: python
 python: 2.7
-install: "pip install flake8"
-script: "flake8 ."
+env:
+  - TOXENV=tests
```

Comment:
Slip-up; will fix in an upcoming patch.  Wasn't intending to run the tests in Travis CI, but definitely meant to retain running flake8 there.